CesiumRasterOverlays/include/CesiumRasterOverlays/CreateRasterOverlayTileProviderParameters.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include <CesiumRasterOverlays/Library.h>
+#include <CesiumRasterOverlays/RasterOverlayExternals.h>
+#include <CesiumUtility/IntrusivePointer.h>
+
+namespace CesiumUtility {
+class CreditSource;
+}
+
+namespace CesiumRasterOverlays {
+
+class RasterOverlay;
+
+/**
+ * @brief Parameters passed to \ref RasterOverlay::createTileProvider.
+ */
+struct CESIUMRASTEROVERLAYS_API CreateRasterOverlayTileProviderParameters {
+  /**
+   * @brief The external interfaces for use by the raster overlay tile provider.
+   */
+  RasterOverlayExternals externals;
+
+  /**
+   * @brief The overlay that owns this overlay.
```

Comment:
Should this be...?

```suggestion
   * @brief The overlay that owns this tile provider.
```

Or...?

```suggestion
   * @brief The overlay that owns the overlay that created this tile provider.
```